brefv/payloads/foxglove/Imu.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package foxglove;
+
+import "foxglove/Quaternion.proto";
+import "foxglove/Vector3.proto";
+import "foxglove/Header.proto";
+
+message Imu {
+  Header header = 1;
+
+  Quaternion orientation = 2;
+  repeated double orientation_covariance = 3; // array with 9 elements
```

Comment:
Please add same comment on lines 16 and 19.

Even tough I dont really understand this. The quaternion has length 4 whereas angular_velocity and linear_acceleration has length 3. But all of them have covariance matrices as 3x3. Do you have any ideas/explanations for this? (Yes, I do see that it is the same as in the ROS message definition)

Reply:
Fixed! 

As to why they choose to use it that way I could only guess - I also had to look up what a Quaternion was. Maybe Martin could provide some insight, he's the physics guy after all!